prusti-specs/src/specifications/preparser.rs

```diff
@@ -119,9 +119,9 @@ impl ParserStream {
                 if punct.as_char() != c {
                     return false;
                 }
-                if i + 1 < operator.len() && punct.spacing() != Spacing::Joint {
-                    return false;
-                }
+                // if i + 1 < operator.len() && punct.spacing() != Spacing::Joint {
+                //     return false;
+                // }
```

Comment:
Better to delete or add a comment explaining why the code is disabled.

Reply:
Commented on that one.